src/routes/(console)/apply-credit/+page.svelte

```diff
@@ -189,6 +189,18 @@
     $: selectedOrg = $organizationList?.teams?.find(
         (team) => team.$id === selectedOrgId
     ) as Organization;
+
+    function getNewBillingPlan(organization: Organization): BillingPlan {
+        if (organization?.billingPlan === BillingPlan.SCALE) {
+            return BillingPlan.SCALE;
+        } else if (campaign?.plan) {
+            return campaign.plan;
+        } else {
+            return BillingPlan.PRO;
+        }
+    }
+
+    $: billingPlan = getNewBillingPlan(selectedOrg);
```

Comment:
@ArmanNik and @TorstenDittmann might now more on Svelte best practices, but I don't think we need to create a new function for this. You can put an if statement directly here like:

https://github.com/appwrite/console/blob/4145174c81d72ec46f62b11538e3cf2392674dfc/src/lib/layout/logs.svelte#L66-L68

Reply:
For now I've changed it to a ternary operation. The if statement way didn't seem to do the trick correctly (perhaps @TorstenDittmann or @ArmanNik can point out later on what I'm doing wrong there haha).